python/ray/serve/api.py

```diff
@@ -473,6 +473,7 @@ def run(
     Returns:
         RayServeSyncHandle: A handle that can be used to call the application.
     """
+    # trigger tests
```

Comment:
This was to trigger release smoke tests to get an early signal since the actual release tests take a while to kick off, will remove.